### PR TITLE
GTCHeck: 180 Files, fix confusions and typos.

### PR DESCRIPTION
**Pull request by JKamlah**
New fixes for confusions s-->ſ and typos.

### CODE REVIEW

File: gt/train/ONB_aze_18950706_7/ONB_aze_18950706_7.jpg_tl_248.gt.txt [hunk 1]
```diff
@@ -1 +1 @@
-Amtliche Schlutzkurse der öffentlichen Wiener Börse am 5. Juli 1895.
\ No newline at end of file
+Amtliche Schlutzkurſe der öffentlichen Wiener Börſe am 5. Juli 1895.
```
Comment (stweil) [inline gt/train/ONB_aze_18950706_7/ONB_aze_18950706_7.jpg_tl_248.gt.txt]: Schlußkurſe?

File: gt/train/ONB_ibn_19110701_009/ONB_ibn_19110701_009.tif_tl_3.gt.txt [hunk 2]
```diff
@@ -1 +1 @@
-Innsbrucker Nachrichten
\ No newline at end of file
+Innſbrucker Nachrichten
```
Comment (stweil) [inline gt/train/ONB_ibn_19110701_009/ONB_ibn_19110701_009.tif_tl_3.gt.txt]: https://github.com/UB-Mannheim/AustrianNewspapers/tree/master/gt/train/ONB_ibn_19110701_009/ONB_ibn_19110701_009.tif_tl_3.png is a bad image, but looks more like "Innsbrucker".

File: gt/train/ONB_nfp_18950706_019/ONB_nfp_18950706_019.tif_tl_121.gt.txt [hunk 3]
```diff
@@ -1 +1 @@
-Losevom Jahre 1864
\ No newline at end of file
+Loſevom Jahre 1864
```
Comment (stweil) [inline gt/train/ONB_nfp_18950706_019/ONB_nfp_18950706_019.tif_tl_121.gt.txt]:
```suggestion
Loſe vom Jahre 1864
```

**Comment by JKamlah** [comment]
Thanks for the hints. I've fixed the transcriptions.

**Pull request merged**